agent/hibinit-agent

```diff
@@ -202,67 +211,53 @@ def find_device_for_file(filename):
 
 
 class SwapInitializer(object):
-    def __init__(self, filename, swap_size, touch_swap, btrfs_enabled, mkswap, swapoff, swapon):
-        self.filename = filename
-        self.swap_size = swap_size
-        self.mkswap = mkswap
-        self.swapoff = swapoff
-        self.swapon = swapon
-        self.touch_swap = touch_swap
-        self.btrfs_enabled = btrfs_enabled
-        self.block_size = 1024
-
-    def do_allocate(self):
-        log("Allocating %d bytes in %s" % (self.swap_size, self.filename))
-        with open(self.filename, 'w+') as fl:
-            fallocate(fl, self.swap_size)
-        os.chmod(self.filename, 0o600)
-
-    def setup_btrfs(self):
-        no_cow = "truncate -s 0 {swapfile} && chattr +C {swapfile}".format(swapfile=self.filename)
-        log("Setting /swap to No Copy On Write")
-        check_call(no_cow, shell=True)
+    def __init__(self, config):
+        self.swap_size = config.swap_mb * 1024 * 1024
```

Comment:
How about having a 1MB as a constant and use it everywhere

Reply:
Good suggestion, will set 1KB as there are multiple use cases of KB as well as MB